src/server/network.ts

```diff
@@ -228,7 +228,7 @@ export class Route extends SdkObject {
     if (body === undefined) {
       if (overrides.fetchResponseUid) {
         const context = this._request.frame()._page._browserContext;
-        const buffer = context.fetchResponses.get(overrides.fetchResponseUid);
+        const buffer = context.fetchRequest.fetchResponses.get(overrides.fetchResponseUid);
```

Comment:
This means I can only use context._request for response interception? I think we should be able to use the global one as well.

Reply:
Will do in a separate PR.